transcom/mymove
# B 18331 add ppm closeout office to move details

[B-18331] populate data in Closeout office field under Move Details

Summary
The QAE and TOO users should be able to see the Closeout office under the Move Details section.


Verification Steps for the Author

These are to be checked by the author.


Verification Steps for Reviewers

These are to be checked by a reviewer.


How to test

1. Login as a QAE role
2. Access a PPM move
3. Verify that the Closeout Office field appears under the Destination ZIP field and that it's populated with data if the Closeout Office exists.  This should be available for both Approved Shipments and Requested Shipments.
4.  Login as a TOO role and repeat the steps above.

Frontend


Backend
